### DISCUSSION TITLE
20221117 no WLAN access point on Pi Zero W

### DISCUSSION THREAD

**Discussion by tertiumsquid** [rern/rAudio#1187] [Bug Reports]
Per title, no WLAN on Pi Zero W using 20221117.  Ifconfig shows nothing but Loopback.

20220904 works.

*edit*  WLAN _does_ in fact work on 20221117 if I manually configure it.  The default access point seems to be what's not working.

**Comment by ceejayemm** [comment]
Did you enable your Wifi connection>  I find the the simplest way to do this with a PI Zero W is to place a file called 'wifi' (with no extension) in the /boot directory ie /boot/wifi which contains:

Interface=wlan0
Connection=wireless
IP=dhcp
ESSID="<nameofyourwificonnection>"
Key=<yourwifinetworkpassword>
Security=wpa

Replace <nameofyourwificonnection> and <yourwifinetworkpassword> with your network details, The file will be deleted after a reboot.  I find this the simplest way to config wifi on any RPi model.

Chris

**Reply by tertiumsquid** [reply]
I had not tried that.  I was expecting the access point to appear, which it does on 20220904.  Manually configuring with the /boot/wifi file does in fact work with 20221117.  I'll change the title accordingly.
